tbconnect/utils.py

```diff
@@ -0,0 +1,20 @@
+from userprofile.models import HealthCheckUserProfile
+
+
+def create_user_profile(
```

Comment:
If this is only used in tests, I would put it in test_utils.py and also move it to the test folder